client/config.go

```diff
@@ -3,6 +3,8 @@ package client
 import (
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"os/user"
```

Comment:
Yeah I have no idea why anyone who would  want to write files to the homedir of a different user would get annoyed enough at the prospect of providing a full path to fork 😄 
I've been wrong before though